packages/ng/libraries/root/src/style/definitions/select/_select-input.scss

```diff
@@ -21,7 +21,6 @@
 
 		&[disabled] {
 			cursor: not-allowed;
-			pointer-events: none;
```

Comment:
We just realised with Lucien that it should be the other way around. We should block pointer-events also for "is-disabled" state. Otherwise we can interact with the disabled select/textfield.